### PR TITLE
Update for CUDA 12.4.0

### PR DESCRIPTION
**Pull request by adibbley**
xref: https://github.com/conda-forge/cuda-feedstock/issues/17

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin , please re-render

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,10 +14,10 @@ build:
...
   run:
-    - cuda-nsight 12.3.101               # [not (aarch64 or win)]
-    - nsight-compute 2023.3.1.1
-    - cuda-nvml-dev 12.3.101
-    - cuda-nvvp 12.3.101                 # [not aarch64]
+    - cuda-nsight 12.4.99               # [not (aarch64 or win)]
+    - nsight-compute 2024.1.0.13
+    - cuda-nvml-dev 12.4.99
+    - cuda-nvvp 12.4.99                 # [not aarch64]
     - cuda-libraries-dev {{ version }}
```
Comment (jakirkham) [inline recipe/meta.yaml:21]: Still need PR: https://github.com/conda-forge/cuda-libraries-dev-feedstock/pull/9

**Comment by jakirkham** [comment]
@conda-forge-admin , please restart CI

**Comment by adibbley** [comment]
Not seeing a Travis build following this PR, triggered one here https://app.travis-ci.com/github/conda-forge/cuda-visual-tools-feedstock/builds/269326604

**Comment by jakirkham** [comment]
Thanks Alex! 🙏

Yeah it is likely the same issue we have seen before

Travis CI says they are escalating our bug internally

**Pull request merged**